include/mlir/Analysis/AffineStructures.h

```diff
@@ -477,7 +477,13 @@ class FlatAffineConstraints {
   /// symbolic operands of vMap should match 1:1 (in the same order) with those
   /// of this constraint system, but the latter could have additional trailing
   /// operands.
-  LogicalResult composeMap(AffineValueMap *vMap);
+  LogicalResult composeMap(const AffineValueMap *vMap);
```

Comment:
But the doc still says that const is eschewed only for the core IR types.
From:
https://github.com/tensorflow/mlir/blob/master/g3doc/UsageOfConst.md
_"The design team since decided to change to a different module, which eschews const entirely for the core IR types:"_

_"Note: This document is only discussing things like const Value* and const Operation*. There is no proposed change for other types, e.g. SmallVector references, the immutable types like Attribute, etc."_